.gitignore

```diff
@@ -4,7 +4,8 @@
 .env
 .DS_Store
 venv/
+.venv/
```

Comment:
It's a good practice to include both `venv/` and `.venv/` in the `.gitignore` file to cover different virtual environment naming conventions. However, ensure that this aligns with your project's actual virtual environment directory name.